.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: c
+sudo: required
+dist: xenial
```

Comment:
it is time to be bionic man!
Directly switch to the last LTS, no?

Reply:
Agreed! Unfortunately Travis's default is Trusty (14), and the latest they support is Xenial (16)
So I already switched to the latest they support :-)

Reply:
:+1: